lib/Mojo/File.pm

```diff
@@ -24,7 +24,7 @@ our @EXPORT_OK = ('path', 'tempdir', 'tempfile');
 
 sub basename { File::Basename::basename ${shift()}, @_ }
 
-sub child { $_[0]->new(@_) }
+sub child { $_[0]->new(${shift()}, @_) }
```

Comment:
I've already voted in favor but can you explain the significance of this line?

Reply:
Totally insignificant. It's an optimization that's already in master actually. 😆

Reply:
It makes `->child` calls *a lot* faster though since it avoids stringification later.